### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV to VA.gov Team - Medical Records (Lab & Test)

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#53878]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Thank you for including heading structure information in your wireframes! Incredibly helpful.
- This all looks really good! I appreciate the thought that's gone into it.

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**

**Should:**
- If possible, on the [landing page](https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/a/DP71nmk) each lab test heading should be unique -- ie. you shouldn't have more than one "Microbiology" heading. Screen reader users frequently navigate the page by pulling up a list of headings and then jumping to the heading for the section they're looking for, and multiple headings with the same name will force them to have to guess-and-check rather than confidently know they're going to the right place. Something like "Microbiology - December 15, 2022" would likely be sufficient.   
   
   If that's not desirable from a design perspective, we can also associate the date information with the heading via `aria-describedby`, so even if the heading is just "Microbiology" a screen reader will announce it as "Microbiology - December 15, 2022." That's a workable back-up plan, but in general we try to avoid using ARIA unless it's really necessary. If there's a way you can make the heading text itself unique that would be ideal.
- You mentioned working on finding a way to control the formatting of the lab results. If technical constraints mean this isn't possible, it is what it is. But if it's possible, I would love to see:
   - a good heading structure
   - semantically-coded lists (rather than just using hyphens to indicate list items)
   - a better way to divide content than long strings of characters (things like `=--=--=--=` aren't great for screen reader users)

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by BobbyBaileyRB** [comment]
We are in the process of ensuring that each lab test heading is unique - this is the desired design perspective. Which includes ensuring that we do not have more than one Microbiology. If this is not possible, we will make sure to add logical Aria tags to the headers - such as your given example "Microbiology - December 15, 2022".

Note, as of now, we don't have control of the reports and notes that get pulled from the database. The long strings of characters (=--=--=--=) is out of our control. But I highly agree with you, this isn't great for screen readers and isn't desirable.
